docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Metrics and targets"
+linkTitle: "Metrics and targets"
+weight: 30
+type: "docs"
+---
+
+For [per revision](./global-vs-per-revision.md) configurations, the `metric` annotation specifies which field in the revision settings is compared against the `target` annotation value.
+
+The `target` annotation determines the value that the Autoscaler tries to maintain for the specified `metric` of every replica of an application. `target` annotations per-revision are "metric agnostic", meaning that the target is simply an integer value, which can be applied for any `metric` type.
+
+For example, if the `metric` annotation of a revision is set to [`concurrency`](./concurrency.md), and the `target` annotation has a value of 10, the Autoscaler attempts to ensure that every replica receives 10 requests at a time.
+
+For [global](./global-vs-per-revision.md) configurations using the default [KPA](./supported-autoscaler-types.md) Autoscaler, there are two keys that can be defined to determine the target value for a specific supported metric type.
+These keys are `container-concurrency-target-default` and `requests-per-second-target-default`, since global KPA configurations only support the `concurrency` and `requests-per-second` (rps) metrics.
+
+Deployments using the [HPA](./supported-autoscaler-types.md) Autoscaler can only configure metrics per revision, and only support the `cpu` metric.
```

Comment:
This is not true. The HPA supports concurrency and rps as well. It supports CPU in addition to that. The KPA only supports the former two.